### PR TITLE
filter red when no result found

### LINKED ISSUE #1936: Filter box and text within it should appear red when no filter is found
**Screenshots of the issue**
filter121

<!—-Where-ever possible add a screenshot of the issue.-—>

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by shubham3211**
#### Short description of what this resolves:
When we apply a filter and search for something the filter box is red when no result is found.

Sample:- https://shubham3211.github.io/google-filter-1/index.html
Fixes #1936

### CODE REVIEW

File: src/backend/templates/tracks.hbs [hunk 1]
```diff
@@ -836,9 +853,27 @@
...
+                    $('.room-filter-info').addClass('filter-not-found');
+                    $('.room-filter-info').css("border","1px dotted red")
+                }
+
             } else {
                 $('#no-results').remove();
 
+                if(trackFilterMode){
+                  $('.track-filter-info').removeClass('filter-not-found');
+                  $('.track-filter-info').css("border","1px dotted black")
```
Comment (agbilotia1998) [inline src/backend/templates/tracks.hbs]: @shubham3211 store ` $('.track-filter-info')` in a variable and use it everywhere.

File: src/backend/templates/tracks.hbs [hunk 2]
```diff
@@ -836,9 +853,27 @@
...
+                } else if(roomFilterMode) {
+                    $('.room-filter-info').removeClass('filter-not-found');
+                    $('.room-filter-info').css("border","1px dotted black")
```
Comment (agbilotia1998) [inline src/backend/templates/tracks.hbs]: Store ` $('.room-filter-info')` in a variable and use it everywhere.

**Comment by shubham3211** [comment]
@agbilotia1998 done :+1:

File: src/backend/templates/rooms.hbs [hunk 3]
```diff
@@ -556,12 +556,35 @@
             }
           });
 
-          // show "no results" if search input value matches zero items
+          var trackFilterInfo = $('.track-filter-info');
+          var roomFilterInfo = $('.room-filter-info');
+
+            // show "no results" if search input value matches zero items
```
Comment (agbilotia1998) [inline src/backend/templates/rooms.hbs]: @shubham3211 please correct the indentation for this comment.

File: src/backend/templates/rooms.hbs [hunk 4]
```diff
@@ -845,9 +868,27 @@
             if ($('.date-filter:visible').length == 0  && searchRooms==1) {
                 $('#no-results').remove();
                 $('.date-list').after("<p style = 'padding-left: 15px; margin-top: 15px; font-weight: bold; color: red' id='no-results'>No matching results found.</p>");
+
+                if(trackFilterMode){
+                  $('.track-filter-info').addClass('filter-not-found');
+                  $('.track-filter-info').css("border","1px dotted red")
+
```
Comment (agbilotia1998) [inline src/backend/templates/rooms.hbs]: Replace `$('.track-filter-info')` with trackFilterInfo.

File: src/backend/templates/rooms.hbs [hunk 5]
```diff
@@ -845,9 +868,27 @@
...
+                } else if(roomFilterMode) {
+                    $('.room-filter-info').addClass('filter-not-found');
+                    $('.room-filter-info').css("border","1px dotted red")
+                }
```
Comment (agbilotia1998) [inline src/backend/templates/rooms.hbs]: Replace `$('.room-filter-info')` with roomFilterInfo

Comment (agbilotia1998) [reply src/backend/templates/rooms.hbs]: Make these changes at all places in all the files.

File: src/backend/templates/rooms.hbs [hunk 6]
```diff
@@ -556,12 +556,35 @@
...
+
           // show "no results" if search input value matches zero items
           if ($('.date-filter:visible').length == 0) {
+
             $('#no-results').remove();
             $('.search-filter:first').after('<p id="no-results" style="color: red">No matching results found.</p>');
+
+            if(trackFilterMode){
+              trackFilterInfo.addClass('filter-not-found');
+              trackFilterInfo.css("border","1px dotted red")
```
Comment (agbilotia1998) [inline src/backend/templates/rooms.hbs]: @shubham3211 can't we add `border: 1px dotted red` in `filter-not-found` class?

**Pull request closed without merging**